internal/extensionserver/extensionserver.go

```diff
@@ -187,6 +187,9 @@ func (s *Server) maybeModifyCluster(cluster *clusterv3.Cluster) {
 		backendRef := httpRouteRule.BackendRefs[i]
 		name := backendRef.Name
 		namespace := aigwRoute.Namespace
+		if backendRef.Priority != nil {
+			endpoints.Priority = *backendRef.Priority
```

Comment:
IIUC this will override `fallback: true/false` option of EG's Backend resource right? Maybe we should document that in the API.